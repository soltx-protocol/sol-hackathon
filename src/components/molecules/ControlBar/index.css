.control-bar {
	display: flex;
	align-items: center;
	justify-content: space-between;
	height: 90px;
	background: rgb(68 72 131 / 60%);
	padding: 0 36px;

	& > .king {
		display: flex;
		align-items: center;

		& > .description {
			font-style: normal;
			font-weight: 400;
			font-size: 16px;
			line-height: 20px;
			margin-left: 10px;
			white-space: pre-line;
			color: #f8f8ff;
		}
	}

	& > .wallet {
		display: flex;
		align-items: center;

		& > .input {
			width: 500px;
			margin-right: 20px;
		}
	}
}
