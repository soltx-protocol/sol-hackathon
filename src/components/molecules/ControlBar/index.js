import React from 'react';
import classnames from 'classnames';

import Input from 'components/atoms/Input';
import Button from 'components/atoms/Button';

import KingIcon from 'images/icon/king.inline.svg';

import styles from './index.css';

const ControlBar = ({ className }) => (
	<div className={classnames(styles.controlBar, className)}>
		<div className={styles.king}>
			<KingIcon />
			<div className={styles.description}>{'Top Asset Owner\nToday'}</div>
		</div>
		<div className={styles.wallet}>
			<Input className={styles.input} placeholder="Insert wallet address" />
			<Button>Generate</Button>
		</div>
	</div>
);

export default ControlBar;
