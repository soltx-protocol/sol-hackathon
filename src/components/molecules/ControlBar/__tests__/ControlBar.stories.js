import React from 'react';
import { storiesOf } from '@storybook/react';

import ControlBar from 'components/molecules/ControlBar';

const stories = storiesOf('molecules/ControlBar', module);

stories.add('__interactive', () => <ControlBar />, {
	redux: {
		data: {
			account: {
				address: '',
				pools: [],
			},
			routing: {
				pathname: '/',
			},
		},
	},
});
