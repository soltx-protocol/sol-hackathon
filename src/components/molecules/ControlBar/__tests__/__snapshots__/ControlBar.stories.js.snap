// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="controlBar"
  >
    <div
      class="king"
    >
      <button
        type="button"
      >
        <test-file-stub />
      </button>
      <div
        class="description"
      >
        Top Asset Owner
Today
      </div>
    </div>
    <div
      class="wallet"
    >
      <div
        class="input input"
      >
        <input
          placeholder="Insert wallet address"
          value=""
        />
      </div>
      <button
        class="button"
        type="button"
      >
        <div>
          Generate
        </div>
      </button>
    </div>
  </div>
</DocumentFragment>
`;
